osmo_jupyter/dataset/source_files.py

```diff
@@ -39,6 +41,12 @@
     "summary_movies",
 ]
 
+VALID_FILETYPES = [".csv", ".mp4", ".gif"]
+
+
+def _check_filepath_is_valid(filepath):
```

Comment:
nit: validity doesn't seem quite like the right thing to be checking here, and "data" seems like a helpful thing to mention 
```suggestion
def _is_data_filepath(filepath):
```

Reply:
+1 I had a similar comment about the module constant name